core/trino-server-rpm/src/main/rpm/preinstall

```diff
@@ -55,10 +55,10 @@ if [ "$java_found" = false ]; then
 +======================================================================+
 |      Error: Required Java version could not be found                 |
 +----------------------------------------------------------------------+
-| Please install JDK 17. On RHEL/CentOS, use java-17-openjdk-devel.    |
+| Please install JDK 21. On RHEL/CentOS, use java-21-openjdk-devel.    |
```

Comment:
Is this package name still correct?

Reply:
I've corrected the whole error section.

Reply:
<img width="674" alt="Screenshot 2023-10-24 at 19 57 34" src="https://github.com/trinodb/trino/assets/66972/d554a513-5df7-4f33-9668-7560286c1eb1">

is it better?

Reply:
Yeah. I like that